Twig/Extension/DisqusExtension.php

```diff
@@ -11,23 +11,20 @@
 
 namespace Knp\Bundle\DisqusBundle\Twig\Extension;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Knp\Bundle\DisqusBundle\Helper\DisqusHelper;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
 
-/**
- *
- */
-class DisqusExtension extends \Twig_Extension
+class DisqusExtension extends AbstractExtension
 {
-    protected $container;
-
     /**
-     * Constructor.
-     *
-     * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+     * @var DisqusHelper
      */
-    public function __construct(ContainerInterface $container)
```

Comment:
The container here was not on accident - it's for performance/laziness. But we *should* change it to use this new "lazy" methodology - https://symfony.com/doc/current/templating/twig_extension.html#creating-lazy-loaded-twig-extensions